Stripe/PublicHeaders/STPCard.h

```diff
@@ -110,6 +110,11 @@ typedef NS_ENUM(NSInteger, STPCardFundingType) {
 /**
  *  The cardholder's address.
  */
+@property(nonatomic, copy, nullable) STPAddress *address;
```

Comment:
if this is inherited from `STPCardParams`, shouldn't it have the same nullability?